pyproject.toml

```diff
@@ -8,14 +8,14 @@ authors = ["Arthur Mesquita Pickcius <arthur.pickcius@gmail.com>"]
 python = "^3.7"
 python-telegram-bot = "^11.1"
 dataset = "^1.1"
-pillow = "^6.1"
+pillow = "^8.0"
```

Comment:
Any specific reason for the updates? Or just to keep them up-to-date?

Reply:
ImageDraw.text anchor parameter was implemented only in pillow 8.0.
[Doc](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.text)